spec/src/lib/course-metadata-api.ts

```diff
@@ -152,16 +152,16 @@ export async function getCourseHomeworkAndQuizInfo(
   return response.data as CourseQuizAndHomeworkInfo;
 }
 
-export async function updateSeriesId(data: {
-  courseId: string;
-  instanceId: string;
-  seriesId: string;
-}) {
-  const res = await axios.post('/api/course-metadata/update-series-id', data, {
-    headers: getAuthHeaders(),
-  });
-  return res.data;
-}
+// export async function updateSeriesId(data: {
```

Comment:
remove this code please. No comment code unless there is a very specific reason. We can always look at git history